src/the-final-count-down/modules/CountDownClock.js

```diff
@@ -0,0 +1,145 @@
+define([
+    "countdown",
+    "modules/MicroEvent",
+    "loDash"
+], function(countdown, MicroEvent, _){
+
+    /**
+     * A wrapper that providers an API for countdown.js
+     *
+     * @param {Date} date
+     * @param {string} mode
+     * @constructor
+     */
+    var CountDownClock = function(date, mode) {
+
+        // The mode for this instance of the clock
+        this.mode = mode || CountDownClock.MODES.COUNT_DOWN;
+
+        // The date we're counting down to
+        this.date = date;
+
+        // bitwise operator containing the units to be displayed, ie: HOURS, SECONDS, etc
```

Comment:
This comment confuses me, I don't know what the type of this variable is supposed to be.